README.md

```diff
@@ -1,2 +1,442 @@
 # FeedTheMonsterH5P
-Feed The Monster frontend web exploration
+<div id="top"></div>
+
+
+
+
+
+
+
+
+
+[![Contributors][contributors-shield]][contributors-url]
+[![Forks][forks-shield]][forks-url]
+[![Stargazers][stars-shield]][stars-url]
+[![Issues][issues-shield]][issues-url]
+
+
+
+
+
```

Comment:
@amitsinghsutara Can we link the English, French, Swahili, and Arabic versions here please?

Reply:
@burrage ,
 Sure will add,
 Thanks a lot for feedback